src/main/java/com/ss/mqtt/broker/handler/packet/in/DisconnetInPacketHandler.java

```diff
@@ -20,6 +20,6 @@ protected void handleImpl(@NotNull UnsafeMqttClient client, @NotNull DisconnectI
             log.error("Disconnect client {} by error reason {}", client, reasonCode);
         }
 
-        client.release().subscribe();
+        client.getConnection().close();
```

Comment:
why did you change release to close?

Reply:
Because of deduplication

Reply:
but when you close this connection, you will not release its client